porfolio.html

```diff
@@ -16,7 +16,41 @@
             <p class="text">John Doe</p>
         </div>
     </div>
- 
+    <div class="headline">
+        <div class="primary-text">Hey There. <br>
+            I’m Fuad
+        </div>
+        <div class="supporting-text">I’am a Software Developer
+        </div>
+        <div class="description">
+             I can help you build a product , feature or<br>
+             website Look through some of my work <br>
+             and experience! If you like what you see <br>
+             and have a project you need coded, <br>
+             don’t hestiate to contact me.
+        </div>
+        <div class="social-media">
+            <button type="button" disabled class="github">
+                <img src="./images/git.png" alt="a link to github" class="git-icon">
+            </button>
+            <button type="button" disabled class="linkedin">
+                <img src="./images/linkedin.png" alt="a link to linkedin" class="linkedin-icon">
+            </button>
+            <button type="button" disabled class="angle-list">
+                <img src="./images/Enabled.png" alt="a link to angle" class="angle-list-icon">
+            </button>
+            <button type="button" disabled class="twitter">
+                <img src="./images/twitter.png" alt="a link to twitter" class="twitter-icon">
+            </button>
+            <button type="button" disabled class="medium">
+                <img src="./images/m.png" alt="a link to medium" class="medium-icon">
+            </button>
+            </div>
```

Comment:
- Kindly use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
- Kindly fix the margins between this icons. It is a requirement that you your app looks like the figma template. 
This is your design
<img width="702" alt="Screenshot 2023-01-16 at 12 45 27" src="https://user-images.githubusercontent.com/48546992/212648475-81eef61b-e789-4200-86f3-a2156906c718.png">
This is the figma template

<img width="370" alt="Screenshot 2023-01-16 at 12 47 23" src="https://user-images.githubusercontent.com/48546992/212648547-77889512-e8bd-4a98-98cc-90bae00b76aa.png">